docker-compose.yaml

```diff
@@ -78,6 +85,8 @@ services:
       - "localhost:host-gateway"
     profiles:
       - local
+    volumes:
+      - ray-pip-cache:/tmp/ray_pip_cache
```

Comment:
Being super pedantic here, but creating files or folders with predicable names under `/tmp` is a security risk. Could we randomize that?

Reply:
Hmmmm... It is inside the container, after all, and for the current session the Ray tmp directories are well known. Furthermore, this is bound within the compose, and specifying a dynamic directory probably needs creating the name outside the compose, passing it as an env var and coordinating with the entrypoint too. If you can think of an easy-ish way to do it, I don't have anything against it.

Reply:
No, I'm okay with that. As I said, I'm being super pedantic...